lighthouse-extension/app/src/popup.js

```diff
@@ -173,85 +201,88 @@ function generateOptionsList(background, selectedCategories) {
     frag.appendChild(createOptionItem(category.title, category.id, isChecked));
   });
 
-  const optionsList = document.querySelector('.options__list');
+  const optionsList = find('.options__list');
   optionsList.appendChild(frag);
 }
 
 /**
  * Initializes the popup's state and UI elements.
- * @param {!Window} background Reference to the extension's background page.
  */
-function initPopup() {
-  getBackgroundPage.then(background => {
-    // To prevent visual hiccups when opening the popup, we default the subpage
-    // to the "running" view and switch to the default view once we're sure
-    // Lighthouse is not already auditing the page. This change was necessary
-    // now that fetching the background event page is async.
-    if (background.isRunning()) {
-      showRunningSubpage();
-    } else {
-      hideRunningSubpage();
+async function initPopup() {
+  chrome.tabs.query({active: true, lastFocusedWindow: true}, function(tabs) {
```

Comment:
is changing the order here deliberate?

Reply:
> is changing the order here deliberate?

yeah, it used to fire off the promise to retrieve the background page and immediately run this query in parallel. Since I switched the `promise.then` to `await`, the query wouldn't run until after the background page had been found and the rest of the init stuff had run. Since at least `siteURL` is used for the pre-populated github message in case of a thrown error, it made sense to make sure it was set before the stuff